oak/server/rust/oak_loader/src/main.rs

```diff
@@ -70,24 +77,73 @@ pub struct Opt {
     introspect_port: u16,
     #[structopt(long, help = "Starts the Runtime without an introspection server.")]
     no_introspect: bool,
+    #[structopt(
+        long,
+        help = "Configuration files to expose to the Oak Application, each in key=filename format."
+    )]
+    config_files: Vec<ConfigEntry>,
+}
+
+/// A specification of a configuration entry as human readable key and a path to a file whose
+/// contents constitutes the actual value.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct ConfigEntry {
+    key: String,
+    filename: String,
+}
+
+impl FromStr for ConfigEntry {
+    type Err = anyhow::Error;
+    fn from_str(v: &str) -> Result<Self, Self::Err> {
+        let parts = v.split("=").collect::<Vec<_>>();
```

Comment:
This seems quite brittle. It might be nice in future to have some better defined parsing.

Reply:
I think this is fine for the foreseeable future, we fully control the file names anyways, so there is no risk that we end up with `=` in the file name itself. But sure, if it does happen we can extend it.

Reply:
Seeing that this is also intended for sensitive data, is there a plan to add labels or some other mechanism to protect the data from untrusted nodes?